src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -1,7 +1,7 @@
 package uk.gov.companieshouse.overseasentitiesapi.validation;
 
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
need to replace the .* with separate imports (CH rules)

Reply:
Intellij does this for me automatically
I am going to modify it